src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -88,9 +105,24 @@ export function PharmacyCard(props: PharmacyProps) {
                   <Link url={props.website}>{props.website}</Link>
                 </Stack.Item>
               </Stack>
+              <Stack>
+                {shouldShowMap ? null : (
+                  <Stack.Item>
+                    <Button
+                      icon={LocationMajor}
+                      onClick={() => {
+                        setShouldShowMap(true);
+                      }}
+                    >
+                      Load map and directions
```

Comment:
This should just say "Load map" (or "Show map?), since we're not actually loading any directions